runelite-client/src/main/java/net/runelite/client/plugins/experiencedrop/XpDropPlugin.java

```diff
@@ -85,20 +91,38 @@ public void onWidgetHidden(WidgetHiddenChanged event)
 		}
 
 		String text = widget.getText();
+		final Stream<Widget> siblingWidgets = Arrays.stream(widget.getParent().getDynamicChildren());
+		final List<Integer> spriteIDs = siblingWidgets.map(Widget::getSpriteId).collect(Collectors.toList());
```

Comment:
Why not just leave this as stream? It will be used only once anyway.

Reply:
I've knocked out the unnecessary List in the middle, thanks for the advice.